instructors.md

```diff
@@ -7,196 +7,248 @@ subtitle: Instructor's Guide
 
 ## Overall
 
-Version control might be the most important topic we teach,
-but Git is definitely the most complicated tool.
-However,
-GitHub presently dominates the open software repository landscape,
-so we have to help novices learn just enough Git
-to feel they can and should learn more on their own.
-
-This is why we don't teach branching:
-while it is a power tool in the hands of a knowledgeable user,
-it is an extra cognitive burden for someone who is new to the idea of version control.
-This is also why we don't get into hashes and commit objects with novices,
-but try instead to convince them that version control is:
-
-1.  a better backup system;
-2.  a better Dropbox; and
-3.  a better way to collaborate than mailing files back and forth.
-
-We close with material on licensing because:
-
-1.  questions about who owns what, or can use what,
-    arise naturally once we start talking about
-    using public services like GitHub to store files;
-    and
-2.  the discussion gives learners a chance to catch their breath
-    after what is often a frustrating couple of hours.
+Version control might be the most important topic we teach, but Git is
+definitely the most complicated tool.  However, GitHub presently dominates the
+open software repository landscape, so we have to help novices learn just
+enough Git to feel they can and should learn more on their own.
 
-## Teaching Notes
+There are many things we don't teach:
+
+- we don't teach branching, because despite being a powerful tool, it is an
```

Comment:
It is redundant to say "there are many things we don't teach:" and then start each bullet point with "we don't teach."  Can it be omitted on each bullet point?

- branching <not sure if the extra reasoning is necessary here>
- Git internals, such as hashes and commit objects